src/enzyme_ad/jax/Passes/LLVMToAffineAccess.cpp

```diff
@@ -466,6 +466,40 @@ struct AffineExprBuilder {
     return failure();
   }
 
+  static bool isStrictlyAffine(AffineExpr expr) {
+
+    if (AffineBinaryOpExpr bOpExpr = dyn_cast<AffineBinaryOpExpr>(expr)) {
+      bool ret_flag = false;
+      switch (bOpExpr.getKind()) {
+      case mlir::AffineExprKind::Add:
+        // do nothing
+        break;
+      case mlir::AffineExprKind::Mul: {
+        bool lhsConst = isa<AffineConstantExpr>(bOpExpr.getLHS());
+        bool rhsConst = isa<AffineConstantExpr>(bOpExpr.getRHS());
```

Comment:
nit (though I don't think this is relevant for GB) 
```suggestion
        bool lhsConst = bOpExpr.getLHS().isSymbolicOrConstant();
        bool rhsConst = bOpExpr.getRHS().isSymbolicOrConstant();
```